### ISSUE TITLE
How do I add a renderer for abbreviations in react-markdown?

### ISSUE THREAD

**Issue by ruchiagarwal75** [remarkjs/react-markdown#369]
Trying to render abbreviations for my markdown that looks like this :

```
Page Intro Description" field
*[Intro]: Here's a big definition. 
```

But there is no way to add an abbreviations renderer. Wondering how can I achieve this?

Note: I also tried using the plugin `remark-abbr`, but it throws an error saying
`Renderer for type `abbr` not defined or is not renderable`

**Comment by ChristianMurphy** [comment]
This would be resolved by #428, which uses `remark-rehype` which `remark-abbr` is designed to work with.

**Comment by wooorm** [comment]
This can be solved outside of this project, hence I’ll close this here.

It could be done by depending on https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-abbr#options, although that still needs to be [updated for the new parser in remark](https://github.com/zestedesavoir/zmarkdown/issues/416), and then specifying a custom renderer.
